Tests/test_content.py

```diff
@@ -921,6 +926,36 @@ def execute_testing(tests_settings, server_ip, mockable_tests_names, unmockable_
             comment = 'The following integrations are skipped and critical for the test:\n {}'. \
                 format('\n- '.join(playbook_skipped_integration))
             add_pr_comment(comment)
+        # Sending proxy metrics to GCP
+        try:
+            storage_client = storage.Client()
+            now = datetime.datetime.now().replace(microsecond=0).isoformat()
+            # each log line will be comprised of the following metrics:
+            # - Date
+            # - Count of successful tests
+            # - Count of failed tests
+            # - Count of successful rerecords
+            # - Count of failed rerecords
+            # - IDs of the playbooks that were rerecorded successfully
+            # - Ids of the playbooks that have failed rerecording
+            new_proxy_line = f'{now},' \
+                             f'{proxy.successful_tests_count},' \
+                             f'{proxy.failed_tests_count},' \
+                             f'{proxy.successful_rerecord_count},' \
+                             f'{proxy.failed_rerecord_count},' \
+                             f'{";".join(proxy.rerecorded_tests)},' \
+                             f'{";".join(proxy.failed_rerecord_tests)}\n'
+            bucket = storage_client.bucket(BUCKET_NAME)
+            # Google storage objects are immutable and there is no way to append to them.
+            # The workaround is to create a new temp file and then compose the log file with the new created file
+            # see here for more info https://cloud.google.com/storage/docs/json_api/v1/objects/compose
+            new_file_blob = bucket.blob(f'{LOCKS_PATH}/{WORKFLOW_ID}.txt')
```

Comment:
saving it where the lock file seems out of order. isn't there a better place for it?